servers/visual/shader_types.cpp

```diff
@@ -78,6 +78,7 @@ ShaderTypes::ShaderTypes() {
 	shader_modes[VS::SHADER_SPATIAL].functions["vertex"].built_ins["MODELVIEW_MATRIX"] = ShaderLanguage::TYPE_MAT4;
 	shader_modes[VS::SHADER_SPATIAL].functions["vertex"].built_ins["INV_PROJECTION_MATRIX"] = constt(ShaderLanguage::TYPE_MAT4);
 	shader_modes[VS::SHADER_SPATIAL].functions["vertex"].built_ins["TIME"] = constt(ShaderLanguage::TYPE_FLOAT);
+	shader_modes[VS::SHADER_SPATIAL].functions["vertex"].built_ins["EYE"] = constt(ShaderLanguage::TYPE_INT);
```

Comment:
Does this require constt?

Reply:
Do you want users to get an error/warning when they try to write to it?

Reply:
Yeah it shouldn't be changed :)